src/Aquifer.Jobs/appsettings.json

```diff
@@ -29,7 +29,7 @@
     },
     "OpenAiTranslation": {
         "DefaultLanguageSpecificPromptAppendixFormatString": "Then translate to {0}.",
-        "HtmlAquiferizationBasePrompt": "You receive HTML and then return the HTML with an altered version of the text therein. Never change the formatting of the HTML elements or attributes. For example, if text has an <em> tag around it or a <span> tag with attributes, that tag and its attributes should stay with the text even if it is simplified or moved within the text. Keep the HTML tags and attributes no matter what. Never add a missing html element (i.e. if something starts with <p> but the end is missing </p>, don't add it). Never remove '&nbsp;'.\n\nImprove the clarity of the provided text and simplify it for a global English audience. Aim for an 8th grade reading level. Add parenthetical explanations if needed. Keep sentences under 25 words and break long paragraphs into shorter ones. Use the active voice whenever natural. Spell out abbreviations, except BC and AD. Never echo the original content, only respond with the alteration.",
+        "HtmlAquiferizationBasePrompt": "You receive HTML and then return the HTML with an altered version of the text therein. Never change the formatting of the HTML elements or attributes. For example, if text has an <em> tag around it or a <span> tag with attributes, that tag and its attributes should stay with the text even if it is simplified or moved within the text. Keep the HTML tags and attributes no matter what. Never add a missing html element (i.e. if something starts with <p> but the end is missing </p>, don't add it). Never remove '&nbsp;'.\n\nImprove the clarity of the provided text and simplify it for a global English audience. Aim for an 8th grade reading level. Add parenthetical explanations if needed. Keep sentences under 25 words. Split long paragraphs into shorter ones (using <p> tags where appropriate). Use the active voice whenever natural. Spell out abbreviations, except BC and AD. Never echo the original content, only respond with the alteration.",
```

Comment:
@jwinston-bn tested this with a long paragraph a good number of times and logged to make sure it was adding the tags correctly.